pom.xml

```diff
@@ -39,7 +39,11 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.4.3</version>
+      <version>[,2.4.9999]</version>
```

Comment:
If you use `2.5)`, I think  `2.5.0-rc1` is matched.
`2.5.0-rc1)` is maybe ok because only 2.5.0-rc1 is published in central.